internal/updater/verify.go

```diff
@@ -11,237 +11,23 @@ import (
 	"golang.org/x/crypto/openpgp"
 )
 
+// gpg --expert --full-generate-key
+// (10) ECC (sign only)
+// (5) NIST P-512 (see https://github.com/golang/crypto/blob/eec23a3978ad/openpgp/packet/public_key.go#L227)
+// 2y
```

Comment:
I'm totally not looking forward to the time where we'll have to rotate that key.

Reply:
Yes ... but I think we can just add another key to the byte slice and it should just work. If I understood the golang.org/x/crypto/openpgp docs correctly.